<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TickerAddDialog</class>
 <widget class="QDialog" name="TickerAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#6000e6</string>
     </property>
     <property name="text">
      <string>Add New Ticker</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color:#6000e6</string>
       </property>
       <property name="text">
        <string>Ticker Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtMaterialTextField" name="in_ticker"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Check">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QtMaterialFlatButton" name="bu_check">
         <property name="text">
          <string>Check</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="progress">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QtMaterialCircularProgress" name="info_progress" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="accepted">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/icons/Icons/check_circle_black_24dp.svg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="denied">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/icons/Icons/highlight_off_black_24dp.svg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QtMaterialFlatButton" name="bu_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtMaterialFlatButton" name="bu_ok">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header>qtmaterialflatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialCircularProgress</class>
   <extends>QWidget</extends>
   <header>qtmaterialcircularprogress.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header>qtmaterialtextfield.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
